app/modules/ads/setup.js

```diff
@@ -178,10 +180,10 @@ export const adsSetup = {
     context.set('services.confiant.enabled', instantConfig.get('icConfiant'));
     context.set('services.durationMedia.enabled', instantConfig.get('icDurationMedia'));
     context.set('services.krux.enabled', adsContext.targeting.enableKruxTargeting
-      && instantConfig.isGeoEnabled('wgAdDriverKruxCountries') && !instantConfig.get('wgSitewideDisableKrux'));
+      && instantConfig.get('icKrux') && !instantConfig.get('wgSitewideDisableKrux'));
```

Comment:
What about `wgSitewideDisableKrux`?

Reply:
@Loczi94 I believe we can remove this part.
```suggestion
      && instantConfig.get('icKrux');
```